### ISSUE TITLE
Enable users to specify (and vote on) shortcuts they would like

### ISSUE THREAD

**Issue by simmo-sg** [insidegui/Sharecuts#33]
There are a load of shortcuts I’d like to create but as a non developer I find the way shortcuts interface gets confusing quickly. It is hard to tell what is possible. 
Eg. “Start FaceTime call if Dad is at home”
Sounds trivial & many components seem to be available in Shortcuts, but I cannot figure out how to create this!

**Comment by esaruoho** [comment]
There is no way to check whether dad is home, @simmo-sg .

**Comment by esaruoho** [comment]
but instead of voting on, it would be good to have a place to leave "please make this shortcut for me" type notes on, and specify email address or some other way of getting in touch with them.
just as long as the shortcut is vetted first, of course.

**Comment by r3volution11** [comment]
I agree @simmo-sg. That will be a great resource. I agree with @insidegui, that the core app itself needs to provide a good service before we can focus on helping others that may need some help. I can't wait for us to get there!
